src/hashutil/mod.rs

```diff
@@ -0,0 +1,14 @@
+
+// TODO : Find proper hash libaray
+// Digest? hashes
+pub trait TempHashContainer {
+    fn init_empty() -> Self;
```

Comment:
If you still find this trait useful, this should probably be a "Default" implementation.

Reply:
Still, I'm not sure if we need a wrapper trait for hash function. Will check this again when I implement [noninteractive_prover_felt_channel.cc](https://github.com/starkware-libs/stone-prover/blob/main/src/starkware/channel/noninteractive_prover_felt_channel.cc)